tests/id15_NXstress/test.py

```diff
@@ -0,0 +1,22 @@
+from id15_NXstress import NXstressFromRaw
+import os
+
+# Determine the absolute directory path of test.py
+dir_path = os.path.abspath(os.path.dirname(__file__))
```

Comment:
All this should be in a function that starts with "test" so that pytest picks it up (see other tests). E.g. call it "test_id15_NXstress_instantiate".